lib/mockito.dart

```diff
@@ -404,3 +531,14 @@ void logInvocations(List<Mock> mocks) {
     print(inv.toString());
   });
 }
+
+/// Should only be used during Mockito testing.
```

Comment:
Could be good to express this in the name so it's easier to catch when someone uses it somewhere else.
`resetMockitoStateOnlyInMockitoTesting()`